src/poliastro/frames.py

```diff
@@ -220,6 +227,78 @@ class PlutoICRS(_PlanetaryICRS):
     body = Pluto
 
 
+def _make_rotation_matrix_from_reprs(start_representation, end_representation):
+    """
+    Return the matrix for the direct rotation from one representation to a second representation.
+    The representations need not be normalized first.
+    """
+    A = start_representation.to_cartesian()
+    B = end_representation.to_cartesian()
+    rotation_axis = A.cross(B)
+    rotation_angle = -np.arccos(
+        A.dot(B) / (A.norm() * B.norm())
+    )  # negation is required
+
+    # This line works around some input/output quirks of Astropy's rotation_matrix()
+    matrix = np.array(rotation_matrix(rotation_angle, rotation_axis.xyz.value.tolist()))
+    return matrix
+
+
+_EARTH_ORBIT_PERPEN_POINT_GCRS = UnitSphericalRepresentation(
+    lon=0 * u.deg, lat=66.5 * u.deg
```

Comment:
This is called the "obliquity" and can be obtained from ERFA, see for example 

https://github.com/astropy/astropy/blob/v3.1.2/astropy/coordinates/builtin_frames/ecliptic_transforms.py#L22-L33

Reply:
So, I will try obtaining it fro ERFA!